mlflow/pipelines/steps/predict.py

```diff
@@ -8,13 +8,14 @@
 from mlflow.pipelines.cards import BaseCard
 from mlflow.pipelines.step import BaseStep
 from mlflow.pipelines.utils.execution import get_step_output_path
+from mlflow.pipelines.utils.step import get_pandas_data_profile
 from mlflow.utils._spark_utils import _get_active_spark_session
 
 _logger = logging.getLogger(__name__)
 
 
 # This should maybe imported from the preprocessing step for consistency
-_INPUT_FILE_NAME = "dataset_preprocessed.parquet"
+_INPUT_FILE_NAME = "preprocessed.parquet"
```

Comment:
Renaming this to be more aligned with other dataset names in the regression template.